### DISCUSSION TITLE
Findings API reference

### DISCUSSION THREAD

**Discussion by borovskimateusz** [secureCodeBox/secureCodeBox#2284] [Q&A] [answered]
Hello :) 
I would try get findings result from SecureCodeBox API i see in docs there are finding endpoint but i didn't see any result for my deployment. 

I would try /finding and /findings naming convention add also job name etc. 
apis/execution.securecodebox.io/v1/namespaces/securecodebox-system/findings 
apis/execution.securecodebox.io/v1/namespaces/securecodebox-system/finding

Could anyone provide valid naming convention for get finding through API? I got scan details through get namespaces/securecodebox-system/scans/<scan_Id> - but there are many information about execution not pure finding. 

Thanks in advance for any help :)

**Comment by J12934** [comment, marked as answer]
Hi @borovskimateusz 
the findings are not stored as custom resources but in an .json file in the S3 bucket. 
So you can't fetch them via the kubernetes api.

You can find links to access the findings from the s3 bucket inside the status of the scan.
